Makefile

```diff
@@ -147,6 +147,11 @@ verify-manifests: manifests
 verify-nested-vendor:
 	./scripts/check-staging-vendor.sh
 
+.PHONY: verify-commits
+verify-commits:
+	echo "$$(env)"
```

Comment:
this is going to be removed, I just want to see what variables we have to work with wrt in openshift-ci

Reply:
https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables is typically a good overview of what environment variables are exposed to test steps.